content/articles/online-machine-learning-with-river-python/index.md

```diff
@@ -0,0 +1,446 @@
+Online machine learning is a type of machine learning in which data becomes available in one at a time in a sequential order. At each step the model is updated until at the end we have a more accurate and robust model.
+
+The data is in motion and keeps on changing over time. Its best suited when we have streaming data, where we want to process one sample of data at a time.
+
+This is different from traditional or offline machine learning where the dataset is available locally and a model is generated at the end after learning from the entire dataset. In offline machine learning, all of our datasets are usually locally available. An overview of the difference between online and offline machine learning is shown.
+
+![Online vs Offline machine learning](/engineering-education/multiclass-text-classification-with-pyspark/offline-vs-online-machine-learning.jpg)
+
+In this tutorial, we shall use [River Python](https://riverml.xyz/latest/) to build our model and simulate streaming data.
+
```

Comment:
I've noticed some instances where you've used 'shall'. This is not correct. Please replace all those instances in your tutorial with the words 'will/must'